actions/src/cla_check/claCheck.js

```diff
@@ -0,0 +1,121 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
```

Comment:
This file should be stored at `actions/src/pull_requests/claCheck.js`

Reply:
DONE